stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/components/ComposerIntegrations.kt

```diff
@@ -20,13 +18,12 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  * @param modifier Modifier for styling.
  */
 @Composable
-internal fun RowScope.DefaultComposerIntegrations(
+internal fun DefaultComposerIntegrations(
```

Comment:
Why isn't this a part of the RowScope anymore? This way the user can no longer affect if the integrations property needs to have a `weight()` for example. Or if it needs to be centered/bottom/top oriented.

Reply:
This is an internal implementation which no longer requires the `RowScope`. The weight can be set externally here:

```
    integrations: @Composable RowScope.() -> Unit = {
        DefaultComposerIntegrations(onAttachmentsClick)
    },
```

Reply:
Alright, yeah good point, thanks for making sure!